CHANGELOG

```diff
@@ -1,3 +1,31 @@
+snowplow-fractribution 0.2.0 (2023-0X-XX)
+---------------------------------------
+## Summary
+This version migrates our models away from the `snowplow_incremental_materialization` and instead move to using the built-in `incremental` with an optimization applied on top. 
```

Comment:
Perhaps add a sentence here along the lines of "Although this package did not contain snowplow incremental models you may have such custom tables in your project." (?) Otherwise some people might be confused as to why all these breaking changes.